pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/simple/PersistentQueue.java

```diff
@@ -39,7 +39,7 @@
      * The connection should not be used concurrently.
      * Currently, this is enforced by synchronizing access to methods that use it.
      */
-    private final Connection connection;
```

Comment:
Keep connection private. Add a public method `getConnection` that returns connection.

Reply:
Done